src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1025,27 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
+            const snapshotTablePairs = _.flow(_.toPairs, _.sortBy(_.first))(snapshotTables)
+            return h(Collapse, {
+              key: snapshotName,
+              buttonStyle: { color: colors.dark(), fontWeight: 600 },
```

Comment:
The `Collapse` component has a built-in bottom margin, which is making this look a little off-center. This is ultimately a problem with `Collapse` itself, since it's not good practice for shared components to have external margins or padding (for this reason). For now, we might want to add `marginBottom: 0` here to cancel it out.